promptmeteo/selector/base.py

```diff
@@ -112,10 +119,14 @@ def load_example_selector(
             self._embeddings,
         )
 
-        self._selector = self.selector(
-            vectorstore=vectorstore,
-            k=self._selector_k,
-        )
+        if self.selector == BalancedSemanticSamplesSelector:
```

Comment:
I think that is better call always with kwargs

Reply:
Hi! In this case, the selectors belong to Langchain framework and the class method from_examples does not acept kwargs arguments. How could I address this then?